explainer.md

```diff
@@ -0,0 +1,92 @@
+# Explainer: MediaCapture-Image
+
+## Authors:
+
+- rijubrata.bhaumik@intel.com
+
+## Participate
+
+- https://github.com/w3c/mediacapture-image/issues
+
+## Introduction
+
+Cameras on client platforms like mobile devices, tablets and laptops are increasingly adding advanced features. In many cases, users do use external cameras when the inbuilt cameras are not delivering the required experience, say on a video call. This specification exposes some of the advanced photography primitives in a standards compliant manner using media track constraints in [MediaDevices.getUserMedia()](https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/getUserMedia
+). For manipulating video related capabilities, current settings and constraints are done via [MediaStreamTrack.applyConstraints()](https://developer.mozilla.org/en-US/docs/Web/API/MediaStreamTrack/applyConstraints). Photo-specific capabilities and current settings can be retrieved via [getPhotoCapabilities()](https://w3c.github.io/mediacapture-image/#dom-imagecapture-getphotocapabilities)/[getPhotoSettings()](https://w3c.github.io/mediacapture-image/#dom-imagecapture-getphotosettings) and configured via takePhoto()'s PhotoSettings argument. The produced image can be in the form of a [Blob](https://www.w3.org/TR/FileAPI/#dfn-Blob) or as a [ImageBitmap](https://html.spec.whatwg.org/multipage/imagebitmap-and-animations.html#imagebitmap).
+
+
+## Goals
+
+The [MediaStream Image Capture API](https://w3c.github.io/mediacapture-image/) has been [implemented in Chrome](https://caniuse.com/#search=imageCapture) since 2017. The specification is an extension of the [Media Capture and Streams](https://w3c.github.io/mediacapture-main/) where we specify advanced features related to camera and photography as a set of property set of capabilities, constraints and settings. Most camera applications are written in native technologies to delight users with advanced use cases like controlling `exposureTime`, `focalDistance`, etc. This specification tries to bridge the gap between what is allowed on a native platform and what is exposed to the Web Platform with respect to camera capabilities. Even more advanced cameras have Pan, Tilt and Zoom support. We support those capabilities also keeping in mind the privacy and security nuances those entail and delight users of applications like video conferencing with use cases not possible before on the Web Platform.
```

Comment:
```suggestion
The [MediaStream Image Capture API](https://w3c.github.io/mediacapture-image/) has been [implemented in Chrome](https://caniuse.com/#search=imageCapture) since 2017. The specification is an extension of the [Media Capture and Streams](https://w3c.github.io/mediacapture-main/) where we specify advanced features related to camera and photography as a set of constrainable properties, with capabilities, constraints and settings. Most camera applications are written in native technologies to delight users with advanced use cases like controlling `exposureTime`, `focalDistance`, etc. This specification tries to bridge the gap between what is allowed on a native platform and what is exposed to the Web Platform with respect to camera capabilities. Even more advanced cameras have Pan, Tilt and Zoom support. We support those capabilities also keeping in mind the privacy and security nuances those entail and delight users of applications like video conferencing with use cases not possible before on the Web Platform.
```

Reply:
Yikes github. This is basically s/set of property set of/set of constrainable properties, with/